bases/br_tesouro_custo_transferencia/br_tesouro_custo_transferencia/table_config.yaml

```diff
@@ -0,0 +1,247 @@
+
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_tesouro_custo_transferencia
+
+table_id: br_tesouro_custo_transferencia
+
+# Título da tabela.
+title: Custos com transferências a estados e municípios do Governo Federal
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description:
+
+# As máximas unidades espaciais que a tabela cobre.
+# Exemplo:
+#   - sa.br
+spatial_coverage: sa.br
+
+# Anos cobertos pela tabela.
+# Preencher como lista de intervalos.
+# Exemplos: ['1995(1)2019'], ['2002(2)2010', '2016', '2020'].
+temporal_coverage: ['2015(1)2022']
+
+# A unidade temporal com qual a tabela é atualizada.
+# Opções em 'https://basedosdados.org/api/3/action/bd_available_options'
+update_frequency: monthly
+
+# Nível de observação da tabela: o que representa cada linha.
+observation_level: Consumo de recursos para produção de bens ou serviços
```

Comment:
modificar para:
```
  - entity: year
    columns:
      - ano_lancamento
  - entity: month
    columns:
      - mes_lancamento

Reply:
Mas tem várias observações por ano/mês, continua a ser o observation_level?

Reply:
Aqui depende muito do dado. Se são transferências municipais, o nível da observação é o id_municipio, se for o estado: sigla_uf...